ovs: Fix build on i686

cathay4t opened the pull request:
Got error when compiling in i686 system:

```
16 | const MAX_RECV_RETRY_COUNT: usize = 4 * 1024 * 1024 * 1024 / BUFFER_SIZE;
   |                                     ^^^^^^^^^^^^^^^^^^^^^^
   evaluation of `ovsdb::json_rpc::MAX_RECV_RETRY_COUNT` failed here
```

Fixed by change it to magic number.

CI test enabled to ensure it never happens again.

ihuguet commented on rust/src/lib/ovsdb/json_rpc.rs:
```diff
@@ -13,7 +13,7 @@ const BUFFER_SIZE: usize = 4096;
 // risking nmstate stuck for ever. We assume the OVSDB returns at most
 // 4 GiB data for a single JSON string.
-const MAX_RECV_RETRY_COUNT: usize = 4 * 1024 * 1024 * 1024 / BUFFER_SIZE;
+const MAX_RECV_RETRY_COUNT: usize = 4 * 1024 * 1024 / BUFFER_SIZE * 1024;
```
This formula works well if the division is even, but not otherwise. Changing the value of BUFFER_SIZE can make it quite inaccurate.

Example with BUFFER_SIZE = 4000: Expected value=1073742, actual result=1073152.

What about:
```
BUFFER_SIZE_KB = 4;
BUFFER_SIZE = BUFFER_SIZE_KB * 1024;
MAX_RECV_RETRY_COUNT = 4 * 1024 * 1024 / BUFFER_SIZE_KB;
```

It's still not exact, but it makes clear that entire KB buffers are required and it can only be inaccurate by 1.

cathay4t replied:
Changed to magic number `1048576`. Since it is a constant with comment, for me it is OK to be a magic number.

cathay4t committed:
ovs: Fix build on i686

Got error when compiling in i686 system:

```
16 | const MAX_RECV_RETRY_COUNT: usize = 4 * 1024 * 1024 * 1024 / BUFFER_SIZE;
   |                                     ^^^^^^^^^^^^^^^^^^^^^^
   evaluation of `ovsdb::json_rpc::MAX_RECV_RETRY_COUNT` failed here
```

Fixed by change it to magic number.

CI test enabled to ensure it never happens again.

Signed-off-by: Gris Ge <fge@redhat.com>
